careers/devops.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: DevOps Engineer
+summary: The Haskell Foundation seeks a passionate DevOps professional to support GHC, related Haskell projects and tooling, and the HF itself.
+postedOn: 2021-12-28
+status: Open
+---
+
+# Job Details
+
+## Responsibilities
+
+* Maintain GitLab and CI infrastructure for GHC
+* Improve CI
+* Expand the CI infrastructure to include base the base Haskell ecosystem
+* Support other Haskell libraries and tools, including HLS (Haskell Language Server)
+* Work with the Haskell Infrastructure Team
+* Support OS packaging of GHC and tools
+* Documentation
+* Improvements and expansion of the GHC performance dashboard
```

Comment:
To me, this bullet requires different expertise than all of the other bullets. This requires a degree of web design skills, as well as tool building. Or maybe integrating with Grafana makes this easy? Then it might be worth describing that the job is about feeding data into Grafana.

I would love it if our hire could do this, but this may be a stretch goal. Happy for other, more knowledgeable people to say that this is routine and is a straightforward expectation of a devops hire.

Reply:
agreed. i think we should put "work with haskell infra team" on down in "additional responsibilities could grow to include" -- they're all things that will be figured out over time. the primary responsibilities are the first bullet points.